salesforce/10.19/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -128,16 +128,95 @@ Changing the username for a configuration prevents the connector from loading th
+
+. Log in to Salesforce.
+. Go to *Setup > Integrations > Change Data Capture* and select a Salesforce object, such as Account.
+. Go to Anypoint Studio and create a Mule app.
+. Drag the *Replay Channel Listener* source to the canvas and in the *Streaming Channel* field, specify `/data/AccountChangeEvent`. For a complete list of channel names, refer to https://developer.salesforce.com/docs/atlas.en-us.change_data_capture.meta/change_data_capture/cdc_subscribe_channels.htm[Subscription Channels]. 
+. Drag a *Logger* component to the canvas to show the payload so you can see the received message on the console, for example:
++
+[source,xml,linenums]
+----
+<flow name="ytaoka-salesforce-replaychannelFlow" doc:id="01d0fd5c-f777-4eda-a167-a931ef240f65" >
+		<salesforce:replay-channel streamingChannel="/data/AccountChangeEvent" replayOption="ONLY_NEW" doc:name="Replay channel" doc:id="c036e7c5-86ed-4904-ae34-185ea42319e9" config-ref="Salesforce_Config" replayId="-1" autoReplay="true"/>
+		<logger level="INFO" doc:name="Logger" doc:id="978f0aad-ab09-4910-bc54-a7c3dcc5935c" message="#[payload]"/>
+</flow>
+----
+. Log in to Salesforce and update any Account record, for example, update the name of the Account to `TestName2`. The console appears as follows:
++
+[source,console]
+----
+INFO  2019-11-12 08:17:27,496 [[MuleRuntime].cpuLight.05: [sample].sampleFlow.CPU_LITE @14741f50] [event: xxx] 
+org.mule.runtime.core.internal.processor.LoggerMessageProcessor: {data={schema=CEjkFTwpfASSecY9UGNoOg, 
+payload={LastModifiedDate=2019-11-11T23:17:30.000Z, ChangeEventHeader={commitNumber=10743571519745, commitUser=0050o00000XTesxAAD, 
+sequenceNumber=1, entityName=Account, changeType=UPDATE, 
+changedFields=[Ljava.lang.Object;@4f738b9d, changeOrigin=com/salesforce/api/soap/47.0;client=SfdcInternalAPI/, 
+transactionKey=0002463d-1e88-1d80-5638-15c821f06b79, commitTimestamp=1573514251000, recordIds=[Ljava.lang.Object;@6e812151}, 
+Name=TestName2}, 
+event={replayId=1065378}}, channel=/data/AccountChangeEvent}
+----
+
+== Working with PushTopic Events (Legacy)
+
+Work with PushTopic events by creating a PushTopic, subscribing to a PushTopic, and replaying PushTopic messages.
+
+NOTE: PushTopic events is a legacy product. Salesforce no longer enhances PushTopic events with new features and provides limited support for this product. Instead of PushTopic events, you can use Change Data Capture events. To learn more about Change Data Capture events, refer to the https://developer.salesforce.com/docs/atlas.en-us.246.0.change_data_capture.meta/change_data_capture/cdc_intro.htm[Change Data Capture Developer Guide] and the https://trailhead.salesforce.com/content/learn/modules/change-data-capture[Change Data Capture Basics] Trailhead module.
```

Comment:
```suggestion
[NOTE]
====
PushTopic events is a legacy product. Salesforce no longer enhances PushTopic events with new features and provides limited support for this product. Instead of PushTopic events, use Change Data Capture events. To learn more about Change Data Capture events, refer to the https://developer.salesforce.com/docs/atlas.en-us.246.0.change_data_capture.meta/change_data_capture/cdc_intro.htm[Change Data Capture Developer Guide^] and the https://trailhead.salesforce.com/content/learn/modules/change-data-capture[Change Data Capture Basics^] Trailhead module.
====
```